etc/disable-common.inc

```diff
@@ -143,3 +143,8 @@ blacklist ${PATH}/xfce4-terminal
 blacklist ${PATH}/xfce4-terminal.wrapper
 blacklist ${PATH}/mate-terminal
 blacklist ${PATH}/mate-terminal.wrapper
+blacklist ${PATH}/lilyterm
+blacklist ${PATH}/pantheon-terminal
+blacklist ${PATH}/roxterm
+blacklist ${PATH}/roxterm-config
+blacklist ${PATH}/terminix
```

Comment:
Thanks for adding it.
My thought was that when urxvtc is blacklisted, urxvtcd is also no danger, so it's not necessary to blacklist it.
But blacklisting also doesn't harm...